spec/features/admin/allocations_spec.rb

```diff
@@ -30,6 +30,28 @@
     end
   end
 
+  describe 'List of Allocations' do
+    let!(:john) {
```

Comment:
It is not that common to use brackets for multi-line blocks, it is mostly used `do..end`